<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="qtable_conf">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>741</width>
     <height>701</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="qcombo_profile">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>16</number>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Default</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="qlabel_conf_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Source File:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_delete">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlabel_conf_label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Profile:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="qline_configuration_file">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_saveas">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_load">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_save">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlabel_conf_label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>140</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Double Click a cell to Edit the value</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="qline_edit_key">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>190</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="qlabel_conf_label_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Key:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="qlabel_conf_label_4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="qline_edit_value">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>240</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="qlabel_conf_label_6">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>300</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Change the current Value to:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="qline_edit_newvalue">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>340</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_browse">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>390</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_apply">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>390</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlabel_conf_label_7">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>460</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Any change will be used only after</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="qlabel_conf_label_8">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>490</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>SAVE and LOAD actions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="qline_row">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="qline_col">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="qbutton_clear">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>390</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <zorder>qline_row</zorder>
  <zorder>qline_col</zorder>
  <zorder>qtable_conf</zorder>
  <zorder>qcombo_profile</zorder>
  <zorder>qlabel_conf_label</zorder>
  <zorder>qbutton_delete</zorder>
  <zorder>qlabel_conf_label_5</zorder>
  <zorder>qline_configuration_file</zorder>
  <zorder>qbutton_saveas</zorder>
  <zorder>qbutton_load</zorder>
  <zorder>qbutton_save</zorder>
  <zorder>qlabel_conf_label_2</zorder>
  <zorder>qline_edit_key</zorder>
  <zorder>qlabel_conf_label_3</zorder>
  <zorder>qlabel_conf_label_4</zorder>
  <zorder>qline_edit_value</zorder>
  <zorder>qlabel_conf_label_6</zorder>
  <zorder>qline_edit_newvalue</zorder>
  <zorder>qbutton_browse</zorder>
  <zorder>qbutton_apply</zorder>
  <zorder>qlabel_conf_label_7</zorder>
  <zorder>qlabel_conf_label_8</zorder>
  <zorder>qbutton_clear</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
